src/components/GroupedCombobox/GroupedComboboxContainer.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { itemType } from '../Combobox/utils/constants';
+import { Picklist } from '../Combobox';
+
+class GroupedComboboxContainer extends Component {
+  static propTypes = {
+    children: PropTypes.func.isRequired,
+    id: PropTypes.string.isRequired,
+    isOpen: PropTypes.bool.isRequired,
+    items: PropTypes.arrayOf(itemType).isRequired,
+    onSelect: PropTypes.func.isRequired,
+    onToggle: PropTypes.func.isRequired,
+    selectedItems: PropTypes.arrayOf(itemType).isRequired,
+  }
+
+  onSelect = (id) => {
+    const { onSelect } = this.props;
+    onSelect(id);
+  }
+
+  renderSwitcher = () => {
+    const {
+      id,
+      isOpen,
+      items,
+      onToggle,
+      selectedItems
+    } = this.props;
+
+    return (
+      <Picklist
```

Comment:
Haven't we deprecated `Picklist` component?

Reply:
Yes and no.`Picklist` currently is a reexport of `BaseCombobox` as intended in the LDS. `Picklist` seems to be the official name of the component. `Picklist` was its own beast back before `Comboboxes` where a thing in the LDS and _that_ impl. has been removed from this project with the last major version

Reply:
Shouldn't we just use it here as `BaseCombobox`? IMHO it will be more clean

Reply:
Yes, I will change it